### PR TITLE
Skip broken champion offer test temporarily

### PR DESCRIPTION
**Pull request by NoelLH**

### CODE REVIEW

File: features/champion-offer.feature [hunk 1]
```diff
@@ -1,3 +1,4 @@
+@skip()
```
Comment (bdsl) [inline features/champion-offer.feature:1]:
```suggestion
# This was failing for reasons that seem to be unrelated to any work we
# have done recently, and I can't immediately see how to fix it, so its skipped for now:
@skip()
```

**Comment by bdsl** [comment]
Added suggestion - but also IMHO we shouldn't be skipping tests unless we have a plan to unskip them soon. If we don't they just become cruft in the test suite.

**Comment by NoelLH** [comment]
> 

@bdsl I certainly would want to prioritise fixing this ASAP. So I think `@skip()`ing is worth it for cleaner diffs.

**Comment by bdsl** [comment]
@NoelLH OK, do you want to make a Jira for it? Or I'm happy to.

**Review by bdsl — APPROVED** [review]

**Pull request merged**